cve_bin_tool/cli.py

```diff
@@ -229,52 +252,70 @@ def main(argv=None):
                           **********************************************
                           """
         LOGGER.warning(warning_nolinux)
+    if args["merge"]:
+        LOGGER.info(
+            "You can use -f --format and -o --output-file for saving merged intermediate reports in a file"
+        )
+        merged_cves = MergeReports(merge_files=args["merge"])
+        if args["input_file"]:
+            LOGGER.warning(
+                "Ignoring -i --input-file while merging intermediate reports"
+            )
+        args["input_file"] = merged_cves.merge_reports()
+        # Creates a Object for OutputEngine
 
-    # Database update related settings
-    # Connect to the database
-    cvedb_orig = CVEDB(
-        version_check=not args["disable_version_check"], error_mode=error_mode
-    )
-
-    # if OLD_CACHE_DIR (from cvedb.py) exists, print warning
-    if os.path.exists(OLD_CACHE_DIR):
```

Comment:
We probably don't want to remove this code yet.  It's been a while but only a few released versions.